src/lara-app/hooks/interactive-api.ts

```diff
@@ -11,11 +12,13 @@ import * as iframePhone from "iframe-phone";
 const experiments = require("../../data/experiments.json") as Experiments;
 
 // NOTE: this is only a partial description of the returned data, containing only the fields we are interested in
+type InitInteractiveMode = "authoring" | "runtime" | "report";
 export interface IInitInteractiveData {
-  mode: "authoring" | "runtime";
+  mode: InitInteractiveMode;
```

Comment:
Just a comment:  I don't see us checking using this field anywhere.  Later on we determine if we are in report mode by looking at `interactiveState`. around line 82:

```

           } else if (data.interactiveState) {
          // report mode

```